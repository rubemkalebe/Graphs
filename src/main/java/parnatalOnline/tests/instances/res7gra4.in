7 4
26 14 17 13 26 9
12 22 25 26 21
23 21 7 29
4 28 22
15 8
6
