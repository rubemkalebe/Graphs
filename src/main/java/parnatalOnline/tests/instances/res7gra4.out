46.000000
0 6, 1 2, 2 5, 3 4, 4 6, 5 6, 
16590
0.013868;0.000000;